modules/ROOT/pages/teradata-vantage-engine-architecture-and-concepts.adoc

```diff
@@ -0,0 +1,118 @@
+* Receives the response from the AMPs and sends it back to the requesting client. 
+
+
+=== BYNET 
+BYNET is a system that enables component communication. The BYNET system provides high-speed bi-directional broadcast, multicast, and point-to-point communication and merge functions. It performs three key functions: coordinating multi-AMP queries, reading data from multiple AMPs, regulating message flow to prevent congestion, and processing platform throughput. These functions of BYNET make Vantage highly scalable and enable Massively Parallel Processing (MPP) capabilities.  
+
+=== Parallel Database Extension (PDE)
+Parallel Database Extension (PDE) is an intermediary software layer positioned between the operating system and the Teradata Vantage database. PDE enables MPP systems to use features such as BYNET and shared disks. It facilitates the parallelism that is responsible for the speed and linear scalability of the Teradata Vantage database.  
+
+=== Access Module Processor (AMP)
+AMPs are responsible for data storage and retrieval. Each AMP is associated with its own set of Virtual Disks (Vdisks) where the data is stored, and no other AMP can access that content in line with the shared-nothing architecture. The functions of AMP are:
+
+* Access storage using Vantage’s Block File System Software  
+* Lock management 
+* Sorting rows 
+* Aggregating columns 
+* Join processing 
+* Output conversion 
+* Disk space management 
+* Accounting 
+* Recovery processing 
+
+[NOTE]
+====
+AMPs in Vantage Enterprise, and the Primary Cluster in the case of VantageCloud Lake, store data in a Block File System (BFS) format on Vdisks. AMPs in Compute Clusters and Compute Worker Nodes on Lake do not have BFS, they can only access data in object storage using the Object File System (OFS). 
+====
+
+=== Virtual Disks (Vdisks)
+These are units of storage space owned by an AMP. Virtual Disks are used to hold user data (rows within tables). Virtual Disks map to physical space on a disk.
+
+=== Node
+A node, in the context of Teradata systems, represents an individual server that functions as a hardware platform for the database software. It serves as a processing unit where database operations are executed under the control of a single operating system. When Teradata is deployed in a cloud, it follows the same MPP, shared-nothing architecture but the physical nodes are replaced with virtual machines (VMs). 
+
+== Teradata Vantage Architecture Concepts
+The concepts below are applicable to Teradata Vantage.
+
+=== Linear Growth and Expandability 
+Teradata is a linearly expandable RDBMS. As the workload and data volume increase, adding more hardware resources such as servers or nodes results in a proportional increase in performance and capacity. Linear Scalability allows for increased workload without decreased throughput.  
+
+image::{dir}/performance_impact_of_adding_components.png[Performance Impact of Adding Components, width=75%]
```

Comment:
I don't understand what this diagram tries to say.

Reply:
This is to show how increase in workload can be adjusted with linear scaling. I believe the text itself is enough to give the idea of the concept and there is no need for this diagram.